packages/apputils-extension/test/palette.spec.ts

```diff
@@ -1,19 +1,23 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+import { CommandRegistry } from '@lumino/commands';
 import { Palette } from '../src/palette';
 import { CommandPalette } from '@lumino/widgets';
 
 describe('palette', () => {
   let palette: Palette;
-  let commandPalette: CommandPalette;
+  let commandPalette: CommandPalette = new CommandPalette({
+    commands: new CommandRegistry()
+  });
 
   beforeEach(() => {
     palette = new Palette(commandPalette);
   });
 
   describe('#ariaLabelsAndRoles', () => {
-    it('command palette should have aria-label and role for accessibility', () => {
+    // Test is currently failing; skipping for now.
+    it.skip('command palette should have aria-label and role for accessibility', () => {
```

Comment:
👍 but let's open an issue for it

Reply:
Opened https://github.com/jupyterlab/jupyterlab/issues/18241